server/src/unifyfs_group_rpc.c

```diff
@@ -54,7 +55,7 @@ static int forward_child_request(void* input_ptr,
     int ret = UNIFYFS_SUCCESS;
 
     /* call rpc function */
-    double timeout_ms = margo_server_server_timeout_msec;
+    double timeout_ms = UNIFYFS_MARGO_SERVER_SERVER_TIMEOUT_MSEC;
```

Comment:
You need to revert this whole file to the current dev version and then add your new `#include unifyfs_rpc_util.h`